File: QTD2.Infrastructure/Model/DIFSurvey/DIFSurveyVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.DIFSurvey
{
    public class DIFSurveyVM
    {
        public int Id { get; set; }
        public string SurveyTitle { get; set; }
        public int PositionId{ get; set; }
        public DateTime StartDate { get; set; }
        public DateTime DueDate { get; set; }
        public string Instructions { get; set; }
        public bool ReleasedToEMP { get; set; }
        public int DevStatusId { get; set; }
        public bool IsActive { get; set; }
        public List<DIFSurveyTaskVM> Tasks { get; set; } = new List<DIFSurveyTaskVM>();
        public List<DIFSurvey_EmployeeVM> Employees { get; set; } = new List<DIFSurvey_EmployeeVM>();
        public string SurveyStatus { get; set; }

        public DIFSurveyVM(int id, string surveyTitle, int positionId, DateTime startDate, DateTime dueDate,string instructions,bool releasedToEMP, int devStatusId, bool isActive, string surveyStatus)
        {
            Id = id;
            SurveyTitle = surveyTitle;
            PositionId = positionId;
            StartDate = startDate;
            DueDate = dueDate;
            Instructions = instructions;
            DevStatusId = devStatusId;
            ReleasedToEMP = releasedToEMP;
            IsActive = isActive;
            SurveyStatus = surveyStatus;
        }
        public DIFSurveyVM(){
            
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/DIFSurvey/DIFSurveyVM.ts
import { DIFSurveyTaskVM } from './DIFSurveyTaskVM'; // Import DIFSurveyTaskVM if it's defined in a separate file
import { DIFSurvey_EmployeeVM } from './DIFSurvey_EmployeeVM';

export class DIFSurveyVM {
  id: string;
  surveyTitle: string;
  positionId: string;
  startDate: Date;
  dueDate: Date;
  instructions: string;
  releasedToEMP: boolean;
  devStatusId: string;
  isActive: boolean;
  tasks: DIFSurveyTaskVM[];
  employees:DIFSurvey_EmployeeVM[];
  surveyStatus:string;
}
